.github/pull_request_template.md

```diff
@@ -0,0 +1,65 @@
+<!--
+Thanks for submitting a pull request! Here are some helpful tips:
+
+* Always create branches on and target the `develop` branch.
+* Run all the tests locally and ensure that they are passing.
+* Run `make format` to ensure that the code is formatted.
+* Run `make check` to ensure that all checks passed successfully.
+* Small commits and contributions that attempt one single goal is preferable.
+* If the idea changes or adds anything functional which will affect users, an 
+AIP discussion is required first on the Aurora forum: 
+https://forum.aurora.dev/discussions/AIPs%20(Aurora%20Improvement%20Proposals).
+* Avoid breaking the public API (namely in engine/src/lib.rs) unless required.
+* If your PR is a WIP, ensure that you enable "draft" mode.
+* Your first PRs won't use the CI automatically unless a maintainer starts.
+If this is not your first PR, please do NOT abuse the CI resources.
+
+Checklist:
+- [ ] I have performed a self-review of my code
+- [ ] I have documented my code, particularly in the hard-to-understand areas
```

Comment:
I think this is shouldn't be the part of this PR, so most likely needs to be rebased